### PR TITLE
Remove reference to non-existing archive for unavailable ezjs_min.0.2.2

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

**Comment by mseri** [comment]
Let’s first check if we can find it in a cache backup

**Comment by avsm** [comment]
It's not in the older opam-0.ocaml.org cache.

File: packages/ezjs_min/ezjs_min.0.2.2/opam [hunk 1]
```diff
@@ -32,11 +32,4 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/ocamlpro/ezjs_min.git"
```
Comment (mseri) [inline packages/ezjs_min/ezjs_min.0.2.2/opam:34]:
```suggestion
dev-repo: "git+https://github.com/ocamlpro/ezjs_min.git"
available: false # The tarball is no longer available
```

**Comment by mseri** [comment]
Thanks, then I guess we don't have an alternative

**Pull request merged**